node_modules/oae-authentication/tests/test-external-strategies.js

```diff
@@ -985,6 +985,74 @@ describe('Authentication', function() {
         });
 
         /**
+         * Test that verifies that the user visibility will be set to private when remote_user is used to populate
```

Comment:
As this isn't an actual test, it should be documented as something along the lines of `Test util that ...`